FRAuth/FRAuth/Callbacks/DeviceBindingCallback.swift

```diff
@@ -0,0 +1,221 @@
+// 
+//  DeviceBindingCallback.swift
+//  FRAuth
+//
+//  Copyright (c) 2022 ForgeRock. All rights reserved.
+//
+//  This software may be modified and distributed under the terms
+//  of the MIT license. See the LICENSE file for details.
+//
+
+
+import Foundation
+/**
+ * Callback to collect the device binding information
+ */
+open class DeviceBindingCallback: MultipleValuesCallback {
+    
+    //  MARK: - Properties
+    
+    /// The userId received from server
+    private(set) var userId: String!
+    /// The userName received from server
+    private(set) var userName: String!
+    /// The challenge received from server
+    private(set) var challenge: String!
+    /// The authentication type of the journey
+    private(set) var deviceBindingAuthenticationType: DeviceBindingAuthenticationType!
+    // The title to be displayed in biometric prompt
+    private(set) var title: String!
```

Comment:
Should that be forced?
Can we not have them without the "!" either some as optional or the rest set with a default value or on the init